frontend/views.py

```diff
@@ -124,11 +124,7 @@ def render_lambda_static_content(lr: LambdaResponse):
 
 
 def react_rendered_view(request):
-    url = request.path
-    querystring = request.GET.urlencode()
-    if querystring:
-        url += f'?{querystring}'
```

Comment:
I originally factored this out into its own function, for use by our new middleware, but then I found out about the handy [`request.get_full_path()`](https://docs.djangoproject.com/en/3.0/ref/request-response/#django.http.HttpRequest.get_full_path) method.

Reply:
cool!